src/docs/index.jsx

```diff
@@ -187,6 +169,38 @@ const Component = createClass({
 		location: any,
 	},
 
+	render() {
+		const { componentName } = this.props.match.params;
+
+		return (
+			<Component
+				componentName={componentName}
+				componentRef={lucidComponents[componentName]}
+				location={location}
+			/>
+		);
+	},
+});
+
+const isReactComponent = value => {
+	try {
+		return (
```

Comment:
what would `throw` here?

Reply:
whoops, the try/catch is old code from an earlier attempt to check if the value was a component. will remove.